avocado_vt/options.py

```diff
@@ -34,6 +34,9 @@
 from virttest.standalone_test import SUPPORTED_TEST_TYPES
 
 
+LOG = logging.getLogger('avocado.vt.options')
```

Comment:
I have not idea how to name this logger is best, so I chose the file name first. Please let me know if have any concerns.